package ru.geekbrains.kotlinnotes.data.errors

class NoAuthException : Throwable()